docker/Dockerfile

```diff
@@ -93,6 +90,8 @@ ADD docker/*.sh /usr/local/bin/
 #RUN addgroup --gid 1024 donor_reporting_portal \
 #    && adduser --disabled-password --ingroup donor_reporting_portal -S donor_reporting_portal
 
+RUN DATABASE_URL='psql://postgres:pass@db:5432/postgres' django-admin collectstatic --noinput
```

Comment:
How come you need the `DATABASE_URL env var? No default in the settings?

Reply:
I guess I can add it there